src/EasyList/TodoServiceDB.cs

```diff
@@ -46,29 +47,23 @@ public void DisplayTodo(Todo todo)
         }
         public void DisplayAllTodo(TodoOrder todoOrder)
         {
+            int DisplayId = 1;
+
             var _todoList = _todoRepository.GetAllTodo(todoOrder)
-                            .Select(x => new { x.Id, x.Label, x.Description, x.Priority, x.Status, CreateDate = x.CreatedDate.DateTime.ToLocalTime() , DueDate = x.DueDate?.DateTime.ToLocalTime() });
+                            .Select(x => new { x.Id, DisplayId = DisplayId++, x.Label, x.Description, x.Priority, x.Status, CreateDate = x.CreatedDate.DateTime.ToLocalTime(), DueDate = x.DueDate?.DateTime.ToLocalTime() });
+
+
 
-            ConsoleTableBuilder
-            .From(_todoList.ToList())
-            .WithCharMapDefinition(
-                    CharMapDefinition.FramePipDefinition,
-                    new Dictionary<HeaderCharMapPositions, char> {
-                        {HeaderCharMapPositions.TopLeft, '╒' },
-                        {HeaderCharMapPositions.TopCenter, '╤' },
-                        {HeaderCharMapPositions.TopRight, '╕' },
-                        {HeaderCharMapPositions.BottomLeft, '╞' },
-                        {HeaderCharMapPositions.BottomCenter, '╪' },
-                        {HeaderCharMapPositions.BottomRight, '╡' },
-                        {HeaderCharMapPositions.BorderTop, '═' },
-                        {HeaderCharMapPositions.BorderRight, '│' },
-                        {HeaderCharMapPositions.BorderBottom, '═' },
-                        {HeaderCharMapPositions.BorderLeft, '│' },
-                        {HeaderCharMapPositions.Divider, '│' },
-                    })
-                .ExportAndWriteLine(TableAligntment.Center);
+            var table = new ConsoleTable(nameof(Todo.Id), nameof(Todo.Label), nameof(Todo.Description), nameof(Todo.Priority), nameof(Todo.Status), nameof(Todo.CreatedDate), nameof(Todo.DueDate));
+
+            _todoList.ToList()
```

Comment:
LGTM! You can remove the .ToList() here tho since a foreach loop will execute the .Select statement